### ISSUE TITLE
Project won't build when using Carthage

### ISSUE THREAD

**Issue by liorlevy888** [sunshinejr/Moya-ModelMapper#47]
I'm getting the following error when building with Carthage versions (6.0.0 and 6.0.0-beta.1):

*** Fetching mapper
*** Fetching Result
*** Fetching Alamofire
*** Fetching ReactiveSwift
*** Checking out Moya-ModelMapper at "6.0.0-beta.1"
*** Downloading RxSwift.framework binary at "37"
*** Checking out Moya at "10.0.0-beta.1"
*** Checking out mapper at "7.3.0"
*** Checking out ReactiveSwift at "2.0.1"
*** Checking out Alamofire at "4.5.1"
*** Checking out Result at "3.2.4"

*** Skipped installing RxSwift.framework binary due to the error:
	"Incompatible Swift version - framework was built with 4.0 (swiftlang-900.0.65 clang-900.0.37) and the local version is 4.0.2 (swiftlang-900.0.69.2 clang-900.0.38)."
*** Checking out RxSwift at "4.0.0-rc.0"
*** xcodebuild output can be found in /var/folders/tw/7hw467y973v8432d55mfsr440000gn/T/carthage-xcodebuild.OBVFMS.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "Mapper" in Mapper.xcodeproj
*** Building scheme "Result-iOS" in Result.xcodeproj
*** Building scheme "ReactiveSwift-iOS" in ReactiveSwift.xcworkspace
*** Building scheme "RxBlocking-iOS" in Rx.xcworkspace
*** Building scheme "RxSwift-iOS" in Rx.xcworkspace
*** Building scheme "RxTests-iOS" in Rx.xcworkspace
*** Building scheme "RxCocoa-iOS" in Rx.xcworkspace
*** Building scheme "Moya" in Moya.xcodeproj
*** Building scheme "RxMoya" in Moya.xcodeproj
*** Building scheme "ReactiveMoya" in Moya.xcodeproj
*** Building scheme "Moya-ModelMapper" in Moya-ModelMapper.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/lior/Desktop/test/Carthage/Checkouts/Moya-ModelMapper/Moya-ModelMapper.xcworkspace -scheme Moya-ModelMapper -configuration Release -derivedDataPath /Users/lior/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=2D9F0ACE-E9BA-48EC-A1E7-BECA93F33B0A -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/lior/Desktop/test/Carthage/Checkouts/Moya-ModelMapper)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/tw/7hw467y973v8432d55mfsr440000gn/T/carthage-xcodebuild.OBVFMS.log

Please advise.
Thank you!

**Comment by pelluch** [comment]
I have the same issue with 6.0.0. Here is the corresponding output from the xcodebuild log:

CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
    cd /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper
    export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/pablo/.rbenv/shims:/Users/pablo/bin:/Users/pablo/Library/Android/sdk/platform-tools:/Users/pablo/Library/Android/sdk/tools:/Users/pablo/Library/Android/sdk/ndk-bundle:/usr/local/sbin:/usr/local/Cellar/coreutils/8.28_1/libexec/gnubin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.1.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name ReactiveMoya_ModelMapper -O -whole-module-optimization -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.1.sdk -target arm64-apple-ios8.0 -g -module-cache-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/ModuleCache -Xfrontend -serialize-debugging-options -embed-bitcode -Xfrontend -report-errors-to-debugger -swift-version 3 -I /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos -F /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos -F /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Carthage/Build/iOS -c -num-threads 4 /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducerAvailability.swift /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducer+ModelMapper.swift /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/Moya-ModelMapper/Response+ModelMapper.swift -output-file-map /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya-ModelMapper-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya_ModelMapper.swiftmodule -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-generated-files.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-own-target-headers.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-project-headers.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos/include -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/DerivedSources/arm64 -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/DerivedSources -emit-objc-header -emit-objc-header-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya_ModelMapper-Swift.h -Xcc -working-directory/Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper

CompileSwift normal arm64
    cd /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -emit-bc /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducerAvailability.swift /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducer+ModelMapper.swift /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/Moya-ModelMapper/Response+ModelMapper.swift -target arm64-apple-ios8.0 -Xllvm -aarch64-use-tbi -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS11.1.sdk -I /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos -F /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos -F /Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Carthage/Build/iOS -g -module-cache-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/ModuleCache -swift-version 3 -serialize-debugging-options -report-errors-to-debugger -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-generated-files.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-own-target-headers.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/ReactiveMoya_ModelMapper-project-headers.hmap -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/BuildProductsPath/Release-iphoneos/include -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/DerivedSources/arm64 -Xcc -I/Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/DerivedSources -Xcc -working-directory/Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper -emit-module-doc-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya_ModelMapper.swiftdoc -serialize-diagnostics-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/SignalProducerAvailability.dia -O -module-name ReactiveMoya_ModelMapper -emit-module-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya_ModelMapper.swiftmodule -emit-objc-header-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/ReactiveMoya_ModelMapper-Swift.h -emit-dependencies-path /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/SignalProducerAvailability.d -num-threads 4 -o /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/SignalProducerAvailability.bc -o /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/SignalProducer+ModelMapper.bc -o /Users/pablo/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0-beta.1/Build/Intermediates.noindex/ArchiveIntermediates/ReactiveMoya-ModelMapper/IntermediateBuildFilesPath/Moya-ModelMapper.build/Release-iphoneos/ReactiveMoya-ModelMapper.build/Objects-normal/arm64/Response+ModelMapper.bc
/Users/pablo/src/test/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducerAvailability.swift:13:12: error: no such module 'Moya_ModelMapper'
    import Moya_ModelMapper
           ^
....

** ARCHIVE FAILED **

The following build commands failed:
	CompileSwift normal arm64
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
(2 failures)

**Comment by josefdolezal** [comment]
Hey @liorlevy888 and @pelluch! Thanks for the issue, I will take a look at it later today and let you know. Sorry for the inconvenience!

**Comment by frederik-jacques** [comment]
Yup I'm having the same issue.

Output of log:
```
CompileSwift normal x86_64
    cd /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/RxMoya-ModelMapper/Single+ModelMapper.swift /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducerAvailability.swift /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/ReactiveMoya-ModelMapper/SignalProducer+ModelMapper.swift /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/RxMoya-ModelMapper/ObservableAvailability.swift /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/Moya-ModelMapper/Response+ModelMapper.swift /Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/RxMoya-ModelMapper/Observable+ModelMapper.swift -target x86_64-apple-ios8.0 -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.1.sdk -I /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Products/Release-iphonesimulator -F /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Products/Release-iphonesimulator -g -module-cache-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/ModuleCache -swift-version 3 -serialize-debugging-options -report-errors-to-debugger -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Moya_ModelMapper-generated-files.hmap -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Moya_ModelMapper-own-target-headers.hmap -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Moya_ModelMapper-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Moya_ModelMapper-project-headers.hmap -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Products/Release-iphonesimulator/include -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/DerivedSources/x86_64 -Xcc -I/Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/DerivedSources -Xcc -working-directory/Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper -emit-module-doc-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Moya_ModelMapper.swiftdoc -serialize-diagnostics-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Single+ModelMapper.dia -O -module-name Moya_ModelMapper -emit-module-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Moya_ModelMapper.swiftmodule -emit-objc-header-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Moya_ModelMapper-Swift.h -emit-dependencies-path /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Single+ModelMapper.d -num-threads 8 -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Single+ModelMapper.o -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/SignalProducerAvailability.o -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/SignalProducer+ModelMapper.o -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/ObservableAvailability.o -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Response+ModelMapper.o -o /Users/frederik/Library/Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/Moya-ModelMapper/6.0.0/Build/Intermediates.noindex/Moya-ModelMapper.build/Release-iphonesimulator/Moya-ModelMapper.build/Objects-normal/x86_64/Observable+ModelMapper.o
/Users/frederik/PROJECTS/the-nerd/sandbox/RpxNew/Private-frameworks/RpxNetworking/Carthage/Checkouts/Moya-ModelMapper/Sources/RxMoya-ModelMapper/Single+ModelMapper.swift:10:8: error: no such module 'RxSwift'
import RxSwift
       ^

** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
	CompileSwift normal x86_64
```

**Comment by josefdolezal** [comment]
Hey guys! I spent quite a time solving the issue today and yesterday, but with no luck 😞. There currently two problems with the package. First of all, the source is Swift 4, however the `.swift-version` configuration for swiftenv is set to 3.1. The second issue is with linking dependencies. The Xcode project is unable to locate target dependencies therefore the build fails.

To be honest, I haven't used carthage for over a half year now and it looks like I completely forget how the setup the project. I quickly checked the configuration and everything looks fine to me.

If there is somebody more competent to fix it go ahead, otherwise I will take a look it during this week.

**Comment by pelluch** [comment]
@josefdolezal, much appreciated. I honestly don't know much about configuring projects with Carthage, but I'll definitely try to take a look when I get some time and see if I can come up with something.

**Comment by sunshinejr** [comment]
Hey guys, quick update about Carthage. I found out 2 bugs on Mapper part, that I fixed and I'm waiting on a merge. After that I will be investigating more to see if everything works correctly. Sorry for the long wait!

**Comment by sunshinejr** [comment]
Okay guys, it's working! I've fixed all the tings that were wrong (lyft/mapper, our carthage setup & .xcodeproj). Current working version is 7.0.0. If you still have problems, please remove your cache (as the original 7.0.0 had problems with Carthage, I had to recreate the tag so the cache will ignore it). The cache in Carthage is in `~/Library/Caches/org.carthage.CarthageKit`.

If there is any problem occurring, please create new issue!
